CHANGELOG.md

```diff
@@ -20,4 +31,4 @@ For detailed information on the tasks carried out during this release, please se
 ](https://github.com/wso2/kubernetes-mi/milestone/1)
 
 
-[v1.2.0.2]: https://github.com/wso2/kubernetes-mi/compare/v1.2.0.1...v1.2.0.2
+[v1.2.0.3]: https://github.com/wso2/kubernetes-mi/compare/v1.2.0.2...v1.2.0.3
```

Comment:
@ThilinaManamgoda let's maintain the commit diffs between other releases also.

e.g.

```
[v1.2.0.3]: https://github.com/wso2/kubernetes-mi/compare/v1.2.0.2...v1.2.0.3
[v1.2.0.2]: https://github.com/wso2/kubernetes-mi/compare/v1.2.0.1...v1.2.0.2
[v1.2.0.1]: https://github.com/wso2/kubernetes-mi/compare/<some-version>...v1.2.0.1
```

Reply:
Done